2016-08/august-24.md

```diff
@@ -0,0 +1,43 @@
+## August 24 ([discuss](https://github.com/webpack/meeting-notes/pull/11))
+
+### Attendees
+
+* [Sean Larkin](http://github.com/thelarkinn)
+* [Tobias Koppers](http://github.com/sokra)
+
+
+#### Attendee Notes:
+Johannes is in Iceland at JSConf Iceland (Good luck!), and Juho was unable to make the meeting today. 
+
+### Agenda
+
+#### Documentation Updates
+
+* We were able to reach [Marco Pai](https://github.com/MarcoPai), however the offer he kindly extended to us for puchase of  webpack.io was too steep. We may consider keeping the existing domain, as the traffic is extremely high. We decided to talk about this after documentation has been a little bit more fleshed out. 
```

Comment:
Have you considered using [js.org](https://js.org)? You get HTTPS, gh-pages, and it's free. Also, don't put too much effort behind the domain name. Google will take people where they need to go. It's NBD. Really.